### PR TITLE
id_ID Translation

### PR DESCRIPTION
**Pull request by mohammadrizki**
Hi, I made several correction for Bahasa Indonesia locale (id_ID).  I am new to git and hope this is the correct way to contribute.
Cheers,

Mohammad

### CODE REVIEW

**Comment by asmecher** [comment]
Thanks, @mohammadrizki! Assigning to @mtub, who handles translations.

File: locale/id_ID/locale.xml [hunk 1]
```diff
@@ -986,18 +1044,18 @@ Validasi mungkin gagal dalam pencarian URL memerlukan form data yang diposting.]
 	<message key="notification.mailListDescription">Masukkan alamat email Anda untuk menerima notifikasi langsung tentang konten baru yang penting dan ditambahkan ke jurnal.</message>
 	<message key="notification.notificationsPublicDescription">Halaman ini menunjukkan update penting yang berkaitan dengan jurnal ini seperti terbitan baru atau pengumuman. Anda bisa berlangganan ke notifikasi ini melalui RSS Feed (dengan meng-klik di gambar sebelah kanan), atau <![CDATA[  <a href="{$emailUrl}">melalui email.</a>]]></message>
 
-	<message key="notification.type.articleSubmitted">Artikel baru, "{$title}", telah diserahkan.</message>
+	<message key="notification.type.articleSubmitted">Artikel baru, "{$param}", telah diserahkan.</message>
```
Comment (mtub) [inline locale/id_ID/locale.xml]: here and in the following lines, the parameter should be $title (was changes recently and apparantly after your commit, sorry...

File: locale/id_ID/admin.xml [hunk 2]
```diff
@@ -56,7 +53,7 @@
 	<message key="admin.mergeUsers"> Gabungkan Para Pengguna</message>
 	<message key="admin.mergeUsers.mergeUser"> Gabungkan Para Pengguna</message> 
 	<message key="admin.mergeUsers.into.description"> Pilih pengguna yang akan menghubungkan kepenulisan pengguna sebelumnya, tugas pengeditan.</message>	<message key="admin.mergeUsers.from.description"> Pilih pengguna (atau beberapa pengguna) untuk digabungkan pada akun pengguna lainnya (contoh, ketika seseorang menggunakan dua akun).</message>	
-<message key="admin.mergeUsers.allUsers"> Semua Pengguna</message>	
-<message key="admin.mergeUsers.confirm"> Apakah Anda menginginkan untuk menggabungkan akun yang diseleksi {$oldAccountCount} akun kedalam akun dengan nama "{$newUsername}"? akun {$oldAccountCount} tidak akan ditampilkan lagi.</message>
+	<message key="admin.mergeUsers.allUsers"> Semua Pengguna</message>
```
Comment (mtub) [inline locale/id_ID/admin.xml]: indentation...

File: locale/id_ID/admin.xml [hunk 3]
```diff
@@ -56,7 +53,7 @@
 	<message key="admin.mergeUsers"> Gabungkan Para Pengguna</message>
 	<message key="admin.mergeUsers.mergeUser"> Gabungkan Para Pengguna</message> 
 	<message key="admin.mergeUsers.into.description"> Pilih pengguna yang akan menghubungkan kepenulisan pengguna sebelumnya, tugas pengeditan.</message>	<message key="admin.mergeUsers.from.description"> Pilih pengguna (atau beberapa pengguna) untuk digabungkan pada akun pengguna lainnya (contoh, ketika seseorang menggunakan dua akun).</message>	
-<message key="admin.mergeUsers.allUsers"> Semua Pengguna</message>	
-<message key="admin.mergeUsers.confirm"> Apakah Anda menginginkan untuk menggabungkan akun yang diseleksi {$oldAccountCount} akun kedalam akun dengan nama "{$newUsername}"? akun {$oldAccountCount} tidak akan ditampilkan lagi.</message>
+	<message key="admin.mergeUsers.allUsers"> Semua Pengguna</message>	
+        <message key="admin.mergeUsers.confirm"> Apakah Anda menginginkan untuk menggabungkan akun yang diseleksi {$oldAccountCount} akun kedalam akun dengan nama "{$newUsername}"? akun {$oldAccountCount} tidak akan ditampilkan lagi.</message>
```
Comment (mtub) [inline locale/id_ID/admin.xml]: indentation: replace spaces with tab

File: locale/id_ID/editor.xml [hunk 4]
```diff
@@ -39,8 +39,8 @@
 	<message key="editor.notifyUsers.inProgress"> Pemberitahuan para pengguna...</message> 
 	<message key="editor.usersNotified"> Para Pengguna Telah Diberitahu
 </message>	
-<message key="editor.notifyUsers.allUsers"> Kirimkan pesan ini ke seluruh pengguna yang terhubung dengan jurnal ini ({$count} pengguna)</message>
-<message key="editor.notifyUsers.interestedUsers"> Kirimkan pesan ini ke seluruh pengguna yang terhubung dengan jurnal ini yang telah memberitahukan bahwa mereka menginginkan berita terbaru ({$count} pengguna)</message> 
+    <message key="editor.notifyUsers.allUsers"> Kirimkan pesan ini ke seluruh pengguna yang terhubung dengan jurnal ini ({$count} pengguna)</message>
```
Comment (mtub) [inline locale/id_ID/editor.xml]: indentation: replace spaces with tab

File: locale/id_ID/editor.xml [hunk 5]
```diff
@@ -39,8 +39,8 @@
...
+    <message key="editor.notifyUsers.interestedUsers"> Kirimkan pesan ini ke seluruh pengguna yang terhubung dengan jurnal ini yang telah memberitahukan bahwa mereka menginginkan berita terbaru ({$count} pengguna)</message>
```
Comment (mtub) [inline locale/id_ID/editor.xml]: indentation: replace spaces with tab

File: locale/id_ID/editor.xml [hunk 6]
```diff
@@ -50,11 +50,11 @@
 	<message key="editor.issues.numArticles">Item</message>
 	<message key="editor.issues.noArticles"> Saat ini tidak ada item yang dijadwalkan untuk isu ini.</message> 
 	<message key="editor.issues.confirmDelete"> Apakah Anda yakin ingin menghapus isu ini secara permanen?</message>	
-<message key="editor.issues.published"> Terpublikasikan</message> 
+    <message key="editor.issues.published"> Terpublikasikan</message>
```
Comment (mtub) [inline locale/id_ID/editor.xml]: indentation: replace spaces with tab (some more lines below...)

File: locale/id_ID/locale.xml (hunk 1 above)
Comment (mtub) [inline locale/id_ID/locale.xml]: context: see #1014

File: locale/id_ID/editor.xml [hunk 7]
```diff
@@ -50,11 +50,11 @@
...
 	<message key="editor.issues.noArticles"> Saat ini tidak ada item yang dijadwalkan untuk isu ini.</message> 
 	<message key="editor.issues.confirmDelete"> Apakah Anda yakin ingin menghapus isu ini secara permanen?</message>	
-<message key="editor.issues.published"> Terpublikasikan</message> 
+    <message key="editor.issues.published"> Terpublikasikan</message> 
 	<message key="editor.issues.unpublished"> Tidak Terpublikasikan</message>	
-<message key="editor.issues.volumeRequired"> Dibutuhkan volume dan harus positif, nilai numeris.</message>
+    <message key="editor.issues.volumeRequired"> Dibutuhkan volume dan harus positif, nilai numeris.</message>
 	<message key="editor.issues.numberRequired"> Dibutuhkan angka dan harus positif, nilai numeris.</message>	
-<message key="editor.issues.yearRequired"> Dibutuhkan tahun dan harus positif, nilai numeris.</message> 
+    <message key="editor.issues.yearRequired"> Dibutuhkan tahun dan harus positif, nilai numeris.</message>
```
Comment (mtub) [inline locale/id_ID/editor.xml]: Indentation, but also: Remove leading space at the beginning of the string?

File: locale/id_ID/admin.xml (hunk 3 above)
Comment (mohammadrizki) [inline locale/id_ID/admin.xml]: - fixed, spaces are replaced with tab
- leading spaces at the beginning of the string are deleted

File: locale/id_ID/locale.xml (hunk 1 above)
Comment (mohammadrizki) [inline locale/id_ID/locale.xml]: - fixed, all $param instances are replaced with $title
- spaces are replaced with tab

**Comment by mtub** [comment]
Thank you, @mohammadrizki - this looks good and is an important update to the id_ID locale. Will merge now. Thanks for contributing, this is very much appreciated.

**Pull request merged**